src/components/Categories.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+
+const Categories = () => {
+  const categories = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    dispatch(checkStatus());
+  };
+  return (
+    <div>
+      <button type="button" onClick={handleClick}>
+        Check Status
+      </button>
+      <p>{categories}</p>
+    </div>
```

Comment:
- [ ] Please kindly make sure your button works as expected, normally when clicking on the `Check Status` button it's supposed to return the `Under construction` text under the button. But this is not the case for you, when clicking on the button nothing happens, please kindly fix that 😉

Reply:
Fixed, thank you.